FAQ.md

```diff
@@ -298,7 +298,7 @@ Name                           | Flag                      | Remark
 ------------------------------ | ------------------------- | ----------
 dfdaemon proxy server port     | dfdaemon --port           | The port should   be in the range of 2000-65535.
 dfget uploader server port     | dfget server --port       | You can use command `dfget server` to start a uploader server before using dfget to download if you don't want to use a random port.
-supernode register port        | supernode --port          | You can use `dfget --node IP:port` to register with the specified supernode register port.
+supernode register port        | supernode --port          | You can use `dfget --node host:port` to register with the specified supernode register port.
```

Comment:
I think supernode api port is better than register port here.

Reply:
It's not suitable to be done in this PR . I think we should do it in a separate PR to give the user a clear cognitive change.